test/compose.js

```diff
@@ -65,4 +65,14 @@ describe('compose', function() {
     assert.deepEqual(g(1, 2, 3), [1, 2, 3]);
   });
 
+  it('returns a curried function', function() {
+    function f(a) { return a + '_END'; }
+    function g(b, c, d) { return [b, c, d].join('_'); }
+    var c = R.compose(f, g);
```

Comment:
> Interesting. I'd thought the alphabetical ordering of `f` and `g` was suggestive of application order

Instead it's probably from the pure mathematical usage like this:

    h(x) = f(g(x)), for all x
    <==>
    h = f ∘ g